mule-deployer/src/main/java/org/mule/tools/client/authentication/model/ConnectedAppCredentials.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Mule ESB Maven Tools
+ * <p>
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * <p>
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.tools.client.authentication.model;
+
+/**
+ * @author Mulesoft Inc.
+ * @since 3.4.0
+ */
+public class ConnectedAppCredentials extends AnypointCredential {
+
+  private final String client_id;
```

Comment:
The code is ok, but I think we should be consistent in the naming convention. If we use SnakeCase everywhere we should use it too 😛